apps/src/code-studio/pd/foorm/editor/components/FoormLoadButtons.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {Button, DropdownButton, MenuItem} from 'react-bootstrap';
+import {
+  setLastSaved,
+  setSaveError,
+  setHasJSONError,
+  setLastSavedQuestions
+} from '../foormEditorRedux';
+
+// This should probably be functional -- no state.
+class FoormLoadButtons extends React.Component {
+  static propTypes = {
+    resetCodeMirror: PropTypes.func,
+    namesAndVersions: PropTypes.array,
+    resetSelectedData: PropTypes.func,
+    onSelect: PropTypes.func,
+    dropdownOptions: PropTypes.array,
+    showCodeMirror: PropTypes.func,
+
+    // populated by redux
+    setLastSaved: PropTypes.func,
+    setSaveError: PropTypes.func,
+    setHasJSONError: PropTypes.func,
+    setLastSavedQuestions: PropTypes.func
+  };
+
+  getFormattedConfigurationDropdownOptions() {
+    return this.props.dropdownOptions.map((dropdownOption, i) => {
+      return this.renderMenuItem(
+        () => this.props.onSelect(dropdownOption['id']),
+        dropdownOption['text'],
+        i
+      );
+    });
+  }
+
+  renderMenuItem(clickHandler, textToDisplay, key) {
+    return (
+      <MenuItem key={key} eventKey={key} onClick={clickHandler}>
+        {textToDisplay}
+      </MenuItem>
+    );
+  }
+
+  // There's gotta be a cleaner way than what I'm doing here
+  // Maybe a general "reset" redux action?
+  initializeEmptyCodeMirror = () => {
```

Comment:
i'd prefer if you didn't add new arrow functions to our react code. imo it is (very minor) tech debt.